layers/+lang/haskell/README.org

```diff
@@ -307,7 +346,7 @@ if it doesn't work, then you can change it with ~SPC e s~.
 the file) but bad coding style and code smell. The HLint checker is called
 *after* the flycheck GHC checker.
 
-HLint can be configured via .hlint.yaml.
+HLint can be configured per project via .hlint.yaml (check Hlint docs for more details).
 
 ** Interactive haskell-mode
 Finally, interactive haskell-mode (~SPC m s b~) also displays errors. These
```

Comment:
Can you double check whether this is still true? Basically standard emacs next error and previous error should work in this buffer too.

Reply:
Hmmm intersting, so I tried it out and what I got is that haskell-goto-prev-error does not work (says No further notes from Haskell compiler) while standard emacs next error and previous error work!
Also, haskell-goto-prev-error does not respond to M-n in my case, it seems like it is not connected to that key binding.

So maybe we should just remove this section completely, the one with haskell interactive mode, as you mentioned?